spec/classes/redis_sentinel_spec.rb

```diff
@@ -87,4 +87,26 @@
     }
   end
 
+  describe 'on Debian Stretch' do
```

Comment:
This should probably be "on Debian Jessie".

Reply:
Fixed! 👍